src/App.js

```diff
@@ -206,13 +216,15 @@ class App extends Component {
     const message = this.createCompilationResultMessage(fileName, result)[activeItem];
     return (
       <div>
-        <Menu tabular widths={3}>
+        <Menu tabular widths={5}>
           <Menu.Item active={activeItem == 'bytecode'} name="bytecode" onClick={this.onClickTab}>bytecode</Menu.Item>
           <Menu.Item active={activeItem == 'bytecode_runtime'} name="bytecode_runtime" onClick={this.onClickTab}>runtime bytecode</Menu.Item>
+          <Menu.Item active={activeItem == 'abi'} name="abi" onClick={this.onClickTab}>abi</Menu.Item>
           <Menu.Item active={activeItem == 'ir'} name="ir" onClick={this.onClickTab}>LLL</Menu.Item>
+          <Menu.Item active={activeItem == 'source_map_asm_list'} name="source_map_asm_list" onClick={this.onClickTab}>asm</Menu.Item>
         </Menu>
         <Segment attached='bottom'>
-          {(activeItem == 'ir') ? this.renderLLL(message) : this.renderBytecode(message)}
+          {(['ir', 'source_map_asm_list'].indexOf(activeItem) + 1) ? this.renderText(message) : this.renderBytecode(message)}
```

Comment:
`['ir', 'source_map_asm_list'].indexOf(activeItem) + 1) ?` 

Help me !!!

Reply:
Maybe
```javascript
['ir', 'source_map_asm_list'].indexOf(activeItem) >= 0
```
 is what you want 👍